var/spack/repos/builtin/packages/py-brotli/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyBrotli(PythonPackage):
+    """Python bindings for the Brotli compression library."""
+
+    homepage = "https://github.com/google/brotli"
+    pypi = "Brotli/Brotli-1.1.0.tar.gz"
+
+    version("1.1.0", sha256="81de08ac11bcb85841e440c13611c00b67d3bf82698314928d0b676362546724")
+
+    # Based on wheel availability on PyPI
```

Comment:
I kinda stopped doing this unless I know for a fact that the library is tightly coupled to the CPython API and things likely won't work in future Python versions. This is the case for numpy/pytorch/TF for example. So feel free to keep doing it, but we may need to relax it in the future.

Reply:
Is removed now.